examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py

```diff
@@ -21,19 +25,19 @@ async def main() -> None:
     subaccount_id = address.get_subaccount_id(index=0)
 
     # prepare trade info
-    market_id = "0xa508cb32923323679f29a032c70342c147c17d0145625922b0ef22e955c844c0"
-    orders = [
-        ProtoMsgComposer.OrderData(
-            market_id=market_id,
-            subaccount_id=subaccount_id,
-            order_hash="0x098f2c92336bb1ec3591435df1e135052760310bc08fc16e3b9bc409885b863b"
-        ),
-        ProtoMsgComposer.OrderData(
-            market_id=market_id,
-            subaccount_id=subaccount_id,
-            order_hash="0x8d4e780927f91011bf77dea8b625948a14c1ae55d8c5d3f5af3dadbd6bec591d"
+    market_id = "0xd0f46edfba58827fe692aab7c8d46395d1696239fdf6aeddfa668b73ca82ea30"
+    orders = []
+    async with grpc.aio.insecure_channel(network.grpc_exchange_endpoint) as channel:
```

Comment:
My example shows that they can compose a batch request for different orders on different markets. They will eventually have to write their own logic to filter which order to cancel before composing this, so looping over the all the orders in a single market like this doesn't demonstrate this capability well.

Reply:
Same as above